### PR TITLE
chore: directory spring cleaning

### PR DESCRIPTION
**Pull request by erunion**
## 🧰 Changes

Well it ain't spring but still a good time to clean up the tree while we're working on a major release.

* [x] Updating our `LICENSE` to 2021 and dropping the `.io` in ReadMe.
* [x] Moves the entire app into a `src/` directory structure.
* [x] Renames the `test/` directory to `__tests__` to match how our other open source libraries are structured.

## 🧬 QA & Testing

* [ ] Tests pass?

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,12 +0,0 @@
-root = true
```
Comment (kanadgupta) [inline .editorconfig:1]: RIP lol

Comment (kanadgupta) [reply .editorconfig:1]: We could probably do this in ReadMe proper as well?

File: bin/rdme [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env node
```
Comment (kanadgupta) [inline bin/rdme:1]: I'm assuming it's fine that this file doesn't have an extension since we have this line?

**Review by kanadgupta — APPROVED** [review]

**Review by rahulhegdee — APPROVED** [review]

File: bin/rdme (hunk 2 above)
Comment (erunion) [reply bin/rdme:1]: yep

File: .editorconfig (hunk 1 above)
Comment (erunion) [reply .editorconfig:1]: Yeah we probably should remove it there if it's still there. Prettier takes care of all this for us now.

**Pull request merged**